### DISCUSSION TITLE
Fermi Copper with a continuous source

### DISCUSSION THREAD

**Discussion by GustiLoesch** [mccode-dev/McCode#1746] [Q&A]
Hi :)

I have a quick question about using a Fermi chopper with a continuous source. 
Is there a way to use the "is first" functionality of the disk chopper component?

Best regards
Augustin

**Comment by willend** [comment]
@GustiLoesch sorry for not seeing your question and responding here! 

I trust that the related [e-mail to the McStas user list](https://mailman2.mcstas.org/pipermail/mcstas-users/2024q4/001548.html) was from you - and your question [was answered OK there](https://mailman2.mcstas.org/pipermail/mcstas-users/2024q4/001549.html)? :-)
